### PR TITLE
Italian 4.0.0

### PR DESCRIPTION
**Pull request by mlocati**
Ref. #224

### CODE REVIEW

File: _locales/it/messages.json [hunk 1]
```diff
@@ -516,13 +527,39 @@
 		"message": "No"
 	},
 	// -------------------- Account options --------------------
+	"about": {
+		"message": "Informazioni su"
+	},
 	"about_name": {
 		"message": "DKIM Verifier",
-		"description": "Name of the add-on"
+		"description": "Nome del componente aggiuntivo"
```
Comment (lieser) [inline _locales/it/messages.json:535]: Note that the `description` property is not visible anywhere in the add-on. Its is meant for translators only, so they get a better context that the text is about.

So unless it makes you live as a translator more easy to have it here in Italian, I personally would not translate them in the future.

Please let me know if you want to leave it as is, or revert the translation. You could also completely remove it, but that would make it probably a little harder to compare it with the english translation, to e.g. see if a string is still missing.

**Review by lieser — APPROVED** [review]

**Comment by lieser** [comment]
Thanks for the quick update. As I said in my comment, please let me know if I should merge it as is, or if you want to update it.

File: _locales/it/messages.json (hunk 1 above)
Comment (mlocati) [reply _locales/it/messages.json]: Let's keep those description in English

**Comment by mlocati** [comment]
I just updated this PR: let's keep the description in English, it's much better and easier to be maintained.

**Pull request merged**